lib/semantics/check-do.h

```diff
@@ -34,6 +36,12 @@ class DoChecker : public virtual BaseChecker {
 
 private:
   SemanticsContext &context_;
+
+  void SayBadLeave(const char *stmtChecked, const char *enclosingStmt,
+      const ConstructNode &) const;
+  void CheckDoConcurrentExit(const char *s, const ConstructNode &) const;
```

Comment:
Either use a name that conveys more information that `s`, or leave it out.

Reply:
Thanks for catching this typo.  I'll remove the ```s```.